WordPress/Classes/WordPress.xcdatamodeld/WordPress 143.xcdatamodel/contents

```diff
@@ -996,6 +996,7 @@
     </entity>
     <entity name="UserSuggestion" representedClassName="UserSuggestion" syncable="YES">
         <attribute name="displayName" optional="YES" attributeType="String" syncable="YES"/>
+        <attribute name="id" optional="YES" attributeType="Integer 64" defaultValueString="0" usesScalarValueType="YES" syncable="YES"/>
```

Comment:
@twstokes Added a `UserSuggestion .id` in the latest core data version model. Do you think I should've created a new version?

The change is very small and it didn't require migration or caused any breaking issues.

Reply:
Thanks for bringing this up @salimbraksa!

> Do you think I should've created a new version?
Looking at past revisions to the core data models, I think we should create a new version. We'll also need to update [MIGRATIONS.md](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/MIGRATIONS.md) with what changed.

Should we update the name from `id` to something like `userSuggestionId` or `suggestionId`? Looking at other entity types it seems we've followed that convention. Let me know your thoughts on that.

Reply:
Good point. The only reason I named it "id" because it's named "ID" in the backend API. But you are right 👍

As an alternative to what you suggested, what do you think of naming it just "userId" ( or "accountId") ? I think this naming better communicates that the attribute represents the same id as as "WPAccount.id".